en/identity-server/next/docs/apis/organization-apis/restapis/organization-user-share.yaml

```diff
@@ -24,6 +24,10 @@ paths:
       description: |
         This API shares one or more users across specified organizations, assigning roles based on the provided policy. The policy defines the sharing scope for each organization, including whether access extends to child organizations.
         
+        > **Important** </br>
+        > If you switch a user from a general sharing policy to a selective one or vice versa, changing it directly might cause access issues. To keep things clear and consistent, first, unshare the user from all organizations, then share them again with the desired organization(s) under the new policy/policies. </br> 
+        > Suppose this step is overlooked and unexpected behavior in user sharing is noticed when new organizations are created. In that case, you can use the `/o/users/unshare-with-all` API to reset shared access before setting up a new user sharing.
```

Comment:
If you overlook this step and notice unexpected user-sharing behavior when new organizations are created, use the `/o/users/unshare-with-all` API to reset access before applying the new sharing policy.